packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -218,15 +220,23 @@ export class CodeFlowLogin {
           }
         })
         .catch(async (error) => {
-          CliCodeLogInstance.error("[Login] getTenantToken acquireTokenSilent : " + error.message);
-          const accountList = await this.msalTokenCache?.getAllAccounts();
-          for (let i=0;i<accountList!.length;++i) {
-            this.msalTokenCache?.removeAccount(accountList![i]);
+          if (error.message.indexOf(MFACode) >= 0) {
+            if (this.showMFA) {
```

Comment:
To show warning only once? What's the side effect if show multiple times?

Reply:
Will look like this.
```
The following tenants require Multi-Factor Authentication (MFA). Use 'teamsfx account login azure --tenant TENANT_ID' to explicitly login to a tenant.
1111111111111
The following tenants require Multi-Factor Authentication (MFA). Use 'teamsfx account login azure --tenant TENANT_ID' to explicitly login to a tenant.
2222222222222
```

Reply:
I see...
Make change to existing work flow as less as possible, e.g., catch and show error in `azureLogin`

Reply:
Moved to azureLogin.